examples/helpers/lis_yosupo.rs

```diff
@@ -0,0 +1,24 @@
+// verification-helper: PROBLEM https://judge.yosupo.jp/problem/longest_increasing_subsequence
+
+use proconio::input;
+use programming_team_code_rust::helpers::lis::Lis;
+
+fn main() {
+    input! {
+        n: usize,
+        a: [i64; n]
+    }
+
+    let mut lis = Lis::default();
+
+    for &elem in &a {
+        lis.push(elem);
+    }
+
+    let idxs = lis.get_lis();
+
+    println!("{}", idxs.len());
+    for idx in idxs {
+        print!("{} ", idx);
+    }
```

Comment:
this is a super nit, but you could a more functional version of this with a `.join()`